packages/repository/src/repository-mixin.ts

```diff
@@ -71,6 +72,32 @@ export function RepositoryMixin<T extends Class<any>>(superClass: T) {
       this.bind(repoKey).toClass(repo);
     }
 
+    /**
+     * Add the dataSource to this application.
+     *
+     * @param dataSource The dataSource to add.
+     * @param name The binding name of the datasource; default to dataSource.name
+     *
+     * ```ts
+     *
+     * const ds: juggler.DataSource = new DataSourceConstructor({
+     *   name: 'db',
+     *   connector: 'memory',
+     * });
+     *
+     * app.dataSource(ds);
+     *
```

Comment:
We should separate the code blocks here, or comment out precede `the datasource can ...` with //

Reply:
The comment is like bellow, isn't it?